src/Integration.UnitTests/VsSessionHostTests.cs

```diff
@@ -32,6 +35,11 @@ public void TestInitialize()
             this.serviceProvider = new ConfigurableServiceProvider(assertOnUnexpectedServiceRequest: false);
             this.sonarQubeService = new ConfigurableSonarQubeServiceWrapper();
             this.stepRunner = new ConfigurableProgressStepRunner();
+            this.solutionBinding = new ConfigurableSolutionBinding();
+
+            ConfigurableVsShell shell = new ConfigurableVsShell();
```

Comment:
One of the tests will create an instance of each of the real service, and one of them needs this to be set.